### ISSUE TITLE
Clarification of time effects (Chapter 13)

### ISSUE THREAD

**Issue by dwitvliet** [matheusfacure/python-causality-handbook#60]
In Chapter 13, it is stated that "_adding a time dummy would control for variables that are fixed across time_". However, isn't it the case that time effects are **fixed within** each time point, but **variable across** time?

**Comment by matheusfacure** [comment]
Good point. I'll take a look to see the best correction

**Comment by matheusfacure** [comment]
I've addressed it in the chapter 22 PR. This issue should be fixed once that PR is merged https://github.com/matheusfacure/python-causality-handbook/pull/57/commits/3a6bc3ca6d57d9faceb3581515030893c995ad55

**Comment by dwitvliet** [comment]
That's great, thank you for addressing it so quickly!

**Comment by matheusfacure** [comment]
Reopening because the PR wasn't merge. I think Git has a feature that it closes the PR automatically once that PR is merged into the main branch.
